cmd/minikube/cmd/service_list.go

```diff
@@ -48,28 +51,43 @@ var serviceListCmd = &cobra.Command{
 			os.Exit(reason.ExSvcUnavailable)
 		}
 
-		var data [][]string
-		for _, serviceURL := range serviceURLs {
-			if len(serviceURL.URLs) == 0 {
-				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, "No node port"})
-			} else {
-				servicePortNames := strings.Join(serviceURL.PortNames, "\n")
-				serviceURLs := strings.Join(serviceURL.URLs, "\n")
+		switch output {
+		case "table":
+			printServicesTable(serviceURLs, co)
+		case "json":
+			printServicesJSON(serviceURLs)
+		}
```

Comment:
There should be a default case that outputs something like:
```
exit.Message(reason.Usage, fmt.Sprintf("invalid output format: %s. Valid values: 'table', 'json'", output))
```

Reply:
Thanks for catching this, fixed it!